### PR TITLE
just added a new file

### PR DESCRIPTION
**Pull request by alakdeba10**
not waiting to learn this editor 😄

### CODE REVIEW

File: _posts/0000-01-66-alakd.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Alak was here
+
+![codercat](https://octodex.github.com/codercat.jpg)
```
Comment (a10hiro) [inline _posts/0000-01-66-alakd.md]: Check the filename since `codercat.jpg` returns 404 error.

File: _posts/0000-01-66-alakd.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
```
Comment (a10hiro) [inline _posts/0000-01-66-alakd.md]: Why don't you put your name?
```suggestion
title: "Welcome to Alak's slide deck!"
```

**Comment by alakdeba10** [comment]
@a10hiro please approve my changes 😄

File: _posts/0000-01-66-alakd.md (hunk 2 above)
Comment (alakdeba10) [reply _posts/0000-01-66-alakd.md]: because I want to belong to the A10 community ... LOL

File: _posts/0000-01-66-alakd.md (hunk 1 above)
Comment (alakdeba10) [reply _posts/0000-01-66-alakd.md]: yiou actually checked!!  I know but did not know what to do

**Pull request merged**